crates/binjs_es6/src/lib.rs

```diff
@@ -55,17 +57,70 @@ pub struct Enrich {
     /// Default: `None` (deactivated), as this is an optional, WIP, optimization.
     pub pure_data_threshold: Option<usize>,
 }
+const_with_str! {
+    const DEFAULT_SCOPES: bool = true;
+    const DEFAULT_LAZY_THRESHOLD: u32 = 0;
+    const DEFAULT_PURE_DATA_THRESHOLD: Option<usize> = None;
+    mod defaults_as_strings;
+}
 
 impl Default for Enrich {
     fn default() -> Self {
         Enrich {
-            scopes: true,
-            lazy_threshold: 0,
-            pure_data_threshold: None,
+            scopes: DEFAULT_SCOPES,
+            lazy_threshold: DEFAULT_LAZY_THRESHOLD,
+            pure_data_threshold: DEFAULT_PURE_DATA_THRESHOLD,
         }
     }
 }
 impl Enrich {
+    pub fn args<'a, 'b>(&self) -> Vec<clap::Arg<'a, 'b>> {
+        use clap::*;
+        vec![
+            Arg::with_name("inject-lazy")
+                .long("inject-lazy")
+                .takes_value(true)
+                .value_name("LEVEL")
+                .help("Rewrite the AST for all functions strictly below level LEVEL. Do nothing if LEVEL = 0")
+                .default_value(defaults_as_strings::DEFAULT_LAZY_THRESHOLD),
+            Arg::with_name("inject-scopes")
+                .long("inject-scopes")
+                .takes_value(true)
+                .help("Rewrite the AST to provide scope information.")
```

Comment:
would be nice to say "(defaults to true)"
(or is it added automatically?)

Reply:
It is added automatically.